xAPI.md

```diff
@@ -414,7 +415,7 @@ examples of activities include a book, an e-learning course, a hike or a meeting
 __Activity Provider (AP)__: The software object that is communicating with 
 the LRS to record information about a learning experience. May be similar to a SCORM 
 package in that it is possible to bundle learning assets with the software object that performs this 
-communication, but an Activity Provider could also be separate from the experience it is reporting about.
+communication, but an Activity Provider could also be separate from the experience it is reporting about. This term is deprecated and will be replaced by ["Statement Provider"](#def_statement_provider) in future editions of the xAPI.
```

Comment:
With the greatest of respect and positivity, I disagree with every part of this :-)

I disagree with your optimism about the lack of confusion. I disagree with your view on how early xAPI is in implementation. I disagree that the essential function of activity providers is not to provide activities - in almost all cases tracking is a secondary function of the tool to providing the activity which is being tracked.